nifi-extension-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/InvokeHTTP.java

```diff
@@ -1257,6 +1274,14 @@ private String getLogString(Map<String, List<String>> map) {
      * Returns a Map of flowfile attributes from the response http headers. Multivalue headers are naively converted to comma separated strings.
      */
     private Map<String, String> convertAttributesFromHeaders(final Response responseHttp) {
+        return convertAttributesFromHeaders(responseHttp, "");
+    }
```

Comment:
This method overload does not seem necessary since it is a `private` method. Recommend consolidating all references to a single `convertAttributesFromHeaders` method.